lib/manager/composer/artifacts.ts

```diff
@@ -73,6 +73,30 @@ function getAuthJson(): string | null {
   return is.emptyObject(authJson) ? null : JSON.stringify(authJson);
 }
 
+function getComposerArguments(config): string {
```

Comment:
Better move this to a `./utils.ts` file as refactor PR, see other comment.

Reply:
Done, see #12443.